src/checkout-buttons/strategies/braintree/braintree-paypal-button-strategy.ts

```diff
@@ -33,39 +34,52 @@ export default class BraintreePaypalButtonStrategy implements CheckoutButtonStra
         }
 
         this._braintreeSDKCreator.initialize(paymentMethod.clientToken);
+        const container = `#${options.containerId}`;
+
+        this._renderButtonsData = {
+            paymentMethod,
+            paypalOptions,
+            container,
+        };
+
+        const that = this;
 
         return Promise.all([
-            this._braintreeSDKCreator.getPaypalCheckout(),
-            this._paypalScriptLoader.loadPaypal(),
+            this._braintreeSDKCreator.getPaypalCheckout(this.renderButtons, that),
```

Comment:
I'm not sure why you need to pass `that/this` to `getPaypalCheckout`. It seems all you want to do is to execute `renderButtons` method in the context of `this`. If that's the case, you can just write something like this:
```ts
this._braintreeSDKCreator.getPaypalCheckout(() => this.renderButtons());
```

Reply:
fixed